mantidproject/mantid
# Miscellaneous updates to the docs

At least the Mesh Plot commit would be very nice to have in the upcoming release. As it is all documentation it should be fine!

**Description of work.**

A handful of small docs updates, one commit for each issue.

Check with the individual issues, but broadly:
Fixes #31366 
- Added more commands example script to visualise 3D mesh plots better 

Fixes #31412 
- An extra line has been added to CreateMDHistoWorkspace to help where users have struggled with the input for Names. As this was only a change in the .cpp file, you may need to add a blank line in ``docs/source/algorithms/CreateMDHistoWorkspace.rst`` for docs-html to pick up the new line!

Fixes #31272 
- Improves and updates the Sliceviewer in the MantidBasicCourse and clarifies the scope of the Core testing for the Sliceviewer

Fixes #30400 
- See bullet points in the issue and check if they are now resolved!

**To test:**

- Build docs-html and dev-docs-html
- Check links, formatting and new images.



*This does not require release notes* because **documentation**




---

#### Reviewer ####

Please comment on the following ([full description](http://developer.mantidproject.org/ReviewingAPullRequest.html)):

##### Code Review #####

- Is the code of an acceptable quality?
- Does the code conform to the [coding standards](http://developer.mantidproject.org/Standards/)?
- Are the unit tests small and test the class in isolation?
- If there is GUI work does it follow the [GUI standards](http://developer.mantidproject.org/Standards/GUIStandards.html)?
- If there are changes in the release notes then do they describe the changes appropriately?

##### Functional Tests #####

- Do changes function as described? Add comments below that describe the tests performed?
- Do the changes handle unexpected situations, e.g. bad input?
- Has the relevant (user and developer) documentation been added/updated?

Does everything look good? Mark the review as **Approve**. A member of `@mantidproject/gatekeepers` will take care of it.
